templates/ShareCarePreview.ss

```diff
@@ -89,6 +89,7 @@
 			<% if $IncludeTwitter %><a href="https://cards-dev.twitter.com/validator" target="_blank">(debug)</a><% end_if %>
 			<span class="separator">|</span> <a href="$GooglePlusShareLink" target="_blank">Google+</a>
 			<% if $IncludePinterest %><span class="separator">|</span> <a href="$PinterestShareLink" target="_blank">Pinterest</a><% end_if %>
+			<% if $IncludeLinkedIn %><span class="separator">|</span> <a href="$LinkedInShareLink" target="_blank">LinkedIn</a><% end_if %>
```

Comment:
Please just remove the `<% if %>` wrapper here, so the LinkedIn share link is always visible.